src/Analyzer.Core.UnitTests/TemplateAnalyzerTests.cs

```diff
@@ -226,6 +226,30 @@ public void FilterRules_ValidConfiguration_NoExceptionThrown()
             TemplateAnalyzer.Create(false).FilterRules(new ConfigurationDefinition());
         }
 
+        [TestMethod]
```

Comment:
Thank you for adding this test. Please also add CLI tests in [CommandLineParserTests.cs](https://github.com/Azure/template-analyzer/blob/main/src/Analyzer.Cli.FunctionalTests/CommandLineParserTests.cs)

Reply:
Done